src/status_im/multiaccounts/update/core.cljs

```diff
@@ -1,17 +1,30 @@
 (ns status-im.multiaccounts.update.core
-  (:require [status-im2.constants :as constants]
-            [utils.re-frame :as rf]
-            [status-im.utils.types :as types]
-            [taoensso.timbre :as log]))
+  (:require [status-im.utils.types :as types]
+            [status-im2.constants :as constants]
+            [taoensso.timbre :as log]
+            [utils.re-frame :as rf]))
 
-(rf/defn send-multiaccount-update
+(rf/defn send-contact-update
   [{:keys [db] :as cofx}]
```

Comment:
`cofx` is not used. Too bad our linter can't detect this due to the usage of the `rf/defn` macro...